src/app/fddev/txn.c

```diff
@@ -164,7 +164,7 @@ txn_cmd_fn( args_t *         args,
   client_cfg->net.ephem_udp_port.lo = (ushort)udpsock->listen_port;
   client_cfg->net.ephem_udp_port.hi = (ushort)(udpsock->listen_port + 1);
   client_cfg->initial_rx_max_stream_data = 1<<15;
-  client_cfg->idle_timeout = 100UL * 1000UL * 1000UL; /* 100 millis */
+  client_cfg->idle_timeout = 5000UL * 1000UL * 1000UL; /* 5000 millis */
```

Comment:
I don't know if we should increase this, as it makes the test failure timeout go from 100 millis to 5 seconds. I kinda like having the command fail instantly